tutorials/audio/audio_effects.rst

```diff
@@ -71,22 +67,23 @@ The compressor has many uses. For example:
     If your goal is to prevent a signal from exceeding a given amplitude
     altogether, rather than to reduce the dynamic range of the signal,
     a :ref:`limiter <doc_audio_buses_limiter>` is likely a better choice
-    than a compressor.
-
+    than a compressor for this purpose. However, applying compression before
+    a limiter is still good practice.
 
 Delay
 ~~~~~
 
-Adds an "echo" effect with a feedback loop. It can be used together
-with *Reverb* to simulate wide rooms, canyons, etc. where sound bounces
-are far apart.
+Digital delay essentially duplicates a signal and repeats it at a specified speed with a volume level that decays for each repeat. Delay is great for simulating the acoustic space of a canyon or large room, where sound bounces have a lot of *delay* between their repeats. This is in contrast to reverb, which has a more natural and blurred sound to it. Using this in conjunction with reverb can create very natural sounding environments!
 
 Distortion
 ~~~~~~~~~~
 
-Makes the sound distorted. Godot offers several types of distortion: *overdrive*,
-*tan* and *bit crushing*. Distortion can be used to simulate sound coming through
+Makes the sound distorted. Godot offers several types of distortion:
+- *Overdrive* sounds like a guitar distortion pedal or megaphone. Sounds distorted with this sound like they're coming through
 a low-quality speaker or device.
+- *Tan* sounds like another interesting flavor of overdrive.
+- *Bit crushing* clamps the amplitude of the signal, making it sound flat and crunchy.
+All three types of distortion can add higher frequency sounds to an original sound, making it stand out better in a mix.
```

Comment:
```suggestion

All three types of distortion can add higher frequency sounds to an original sound, making it stand out better in a mix.
```
Otherwise it will be included in the list